### PR TITLE
Fixes the textures inside mesh files for the RoboCup controller

### PR DESCRIPTION
**Pull request by ysims**
The textures seem to work fine in our worlds, but the ones in the separate mesh proto files can't find the textures when running the RoboCup controller because the `robots` folder doesn't exist. This changes them from being relative to the proto to relative to its own location (mesh folder).

### CODE REVIEW

File: protos/robot/NUgus.proto [hunk 1]
```diff
@@ -146,7 +146,7 @@ PROTO NUgus [
                 children [
                   DEF right_upper_arm NUgus_right_upper_armMesh {
                     robotTexture [
-                      %{='"./robot/NUgus_textures/NUgus_upper_arm_' .. color .. '.jpg"'}%
+                      %{='"../NUgus_textures/NUgus_upper_arm_' .. color .. '.jpg"'}%
```
Comment (JosephusPaye) [inline protos/robot/NUgus.proto:149]: Is the `NUgus_textures` folder autogenerated? Can't seem to find it in `protos/`.

Comment (ysims) [reply protos/robot/NUgus.proto:149]: https://github.com/NUbots/NUWebots/tree/main/protos/robot/NUgus_textures

Comment (JosephusPaye) [reply protos/robot/NUgus.proto:149]: Ah, so it's in `protos/robot`. It's weird how webots does the relative paths. 😅

**Review by JosephusPaye — APPROVED** [review]
LGTM

**Pull request merged**